channel-sender/lib/channel_sender_ex/core/security/channel_authenticator.ex

```diff
@@ -10,16 +10,14 @@ defmodule ChannelSenderEx.Core.Security.ChannelAuthenticator do
   @type channel_ref() :: String.t()
   @type channel_secret() :: String.t()
 
-  @spec create_channel(application(), user_ref()) ::
-          {:error, :no_app} | {channel_ref(), channel_secret()}
+  @spec create_channel(application(), user_ref()) :: {channel_ref(), channel_secret()}
   def create_channel(application, user_ref) do
     {channel_ref, _channel_secret} = credentials = create_channel_data_for(application, user_ref)
     {:ok, _pid} = ChannelSupervisor.start_channel({channel_ref, application, user_ref})
     credentials
```

Comment:
what happen if not found the application?

Reply:
At the moment the app existence is not performed